inst/adlb_grading/atoxgr_sources.R

```diff
@@ -14,3 +14,9 @@ atoxgr_criteria_ctcv5 <- atoxgr_criteria %>%
   dplyr::mutate(GRADE_CRITERIA_CODE = gsub("[\r\n]", " ", GRADE_CRITERIA_CODE))
 
 save(atoxgr_criteria_ctcv5, file = "data/atoxgr_criteria_ctcv5.rda")
+
+atoxgr_criteria_daids <- atoxgr_criteria %>%
+  readxl::read_excel(sheet = "DAIDS") %>%
```

Comment:
@millerg23 I'm going to assume the exclusion of certain parameters/conditions were done on purpose like acidosis and alkalosis which is based on pH levels ?

Reply:
Don't assume anythin Zelos ;-) I meant to go back to the PH ones, to see if its worth adding them. Thanks for reminding me! I just got the first unit test done and passed for DAIDS WBC, Decreased, using the FILTER. So hopefully can get everything done fairly soon. The unit tests may take some time!

Reply:
@zdz2101 I have added Acidosis and Akklalosis to metadata, and added some more unit tests.

Reply:
@zdz2101 I am not sure I undrstand what you mean about "AVAL >= 7.3 and AVAL <= 7.5" ?

Reply:
Nvm, 
![image](https://github.com/pharmaverse/admiral/assets/43018762/86445ce1-d7b3-46a2-943d-16bed762ab1c)
 I must've been confused how they original wrote this in/misinterpreted it